src/argilla_server/_app.py

```diff
@@ -95,7 +95,12 @@ async def redirect_api():
     ]:
         app_configure(app)
 
-    return app
+    if settings.base_url and settings.base_url != "/":
+        _app = FastAPI(docs_url=None, redoc_url=None, openapi_url=None)
+        _app.mount(settings.base_url, app)
+        return _app
+    else:
+        return app
```

Comment:
If we set `settings.base_url` to be `/` by default we don't need to check anything and always mount with `settings.base_url`.